src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1022,6 +1073,12 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             // macro directly with a VirtualKeyModifiers
             const auto shiftEnabled = WI_IsFlagSet(modifiers, static_cast<uint32_t>(VirtualKeyModifiers::Shift));
 
+            if (_TrySendMouseEvent(point, true))

```

Comment:
You won't be able to check for which mouse button is released by using `point.Properties().IsLeftButtonPressed()` when inside `_TrySendMouseEvent` and its related ones for middle and right button during pointer _release_.  You can however use `point.Properties().PointerUpdateKind()` instead! Check out the definition here: [PointerUpdateKind](https://docs.microsoft.com/en-us/uwp/api/windows.ui.input.pointerupdatekind). It's an enum so maybe it could simplify the code in your `_TrySendMouseEvent` function too.